sudo.cpp

```diff
@@ -278,10 +278,10 @@ void Sudo::child()
             break;
     }
 
-    std::unique_ptr<char const *[]> params{new char const *[params_cnt]};
+    auto params = std::make_unique<char const *[]>(params_cnt);
     const char ** param_arg = params.get() + 1;
-
-    std::string program = backendName().toLocal8Bit().data();
+    
```

Comment:
Please, avoid trailing spaces.

Reply:
fix it https://github.com/lxqt/lxqt-sudo/pull/219/commits/bf85b6df9c7129b839f8f2e649a5525909ab22b5